snippets/csharp/VS_Snippets_CLR/CatchException/CS/catchexception2.cs

```diff
@@ -10,7 +10,7 @@ public static void Main()
         {
             StreamReader sr = File.OpenText("data.txt");
             Console.WriteLine("The first line of this file is {0}", sr.ReadLine());
-	    sr.Close();
+	        sr.Close();
         }
         catch (Exception e)
```

Comment:
catching Exception is a worst practice, so we wouldn't want to illustrate it. In this case, the constructor and ReadLine method can throw any of three exceptions:
- FileNotFoundException
- DirectoryNotFoundException
- IOException